tgui/packages/tgui/interfaces/TGUITemplate.tsx

```diff
@@ -0,0 +1,102 @@
+import { classes } from 'common/react';
+import { useBackend, useLocalState } from '../backend';
+import { Input, Button, Stack, Box } from '../components';
+import { Window } from '../layouts';
+
+type TGUITemplateData = {
+  options: string[];
+  ui_message: string;
+  current_option: string;
+  something_static: string;
+};
+
+export const TGUITemplate = (props, context) => {
+  const { act, data } = useBackend<TGUITemplateData>(context);
+  const { options, ui_message, current_option, something_static } = data; // not used in this section, but remained here to remind
+
+  return (
+    <Window width={400} height={400} title="TGUI Template">
```

Comment:
add a `theme="ntos"` or something for example purposes as well maybe. or a button that switches the theme to show how useLocalState works.